<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">



<title>Posterdown.knit</title>







<!--
Font-awesome icons ie github or twitter
-->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/brands.css" integrity="sha384-n9+6/aSqa9lBidZMRCQHTHKJscPq6NW4pCQBiMmHdUCvPN8ZOg2zJJTkC7WIezWv" crossorigin="anonymous">

<!--
Google fonts api stuff
-->
<link href='https://fonts.googleapis.com/css?family=Arial' rel='stylesheet'>
<link href='https://fonts.googleapis.com/css?family=Arial' rel='stylesheet'>

<!--
Here are the required style attributes for css to make this poster work :)
-->
<style>
@page {
size: 841mm 1189mm;
margin: 0;
padding: 0;
}
body {
margin: 0px;
padding: 0px;
width: 841mm;
height: 1189mm;
text-align: justify;
font-size: 45px;
line-height: 1.05;
}
/* RMarkdown Class Styles */
/* center align leaflet map,
from https://stackoverflow.com/questions/52112119/center-leaflet-in-a-rmarkdown-document */
.html-widget {
margin: auto;
position: sticky;
margin-top: 2cm;
margin-bottom: 2cm;
}
.leaflet.html-widget.html-widget-static-bound.leaflet-container.leaflet-touch.leaflet-fade-anim.leaflet-grab.leaflet-touch-drag.leaflet-touch-zoom {
position: sticky;
width: 100%;
}
pre.sourceCode.r {
background-color: #dddddd40;
border-radius: 4mm;
padding: 4mm;
width: 75%;
margin: auto;
margin-top: 1em;
margin-bottom: 1em;
/* align-items: center; */
}
code.sourceCode.r{
background-color: transparent;
font-size: 20pt;
border-radius: 2mm;
}
code {
font-size: 25pt;
font-family: monospace;
background-color: #43384d24;
color: #6C5B7B;
padding: 1.2mm;
line-height: 1;
border-radius: 2mm;
}
caption {
margin-bottom: 10px;
font-size: 20pt;
font-style: italic;
}

tbody tr:nth-child(odd) {
    background-color: #43384d20;
}
.table>thead>tr>th, .table>tbody>tr>th, .table>tfoot>tr>th, .table>thead>tr>td, .table>tbody>tr>td, .table>tfoot>tr>td{
  border-spacing: 0;
  font-size: 40%;
  border-style: none;
  padding-top: 15px;
  padding-bottom: 15px;
  padding-right: 1em;
  padding-left: 1em;
  line-height: 1em;
}
table {
  margin: auto;
}
th {
  padding-left: 5mm;
  padding-right: 5mm;
}
.caption {
font-size: 20pt;
font-style: italic;
padding-top: 0;
}
.references {
font-size: 18px;
line-height: 90%;
}
/* Create three unequal columns that floats next to each other */
.column {
float: left;
padding: 0px;
}
.outer {
width: 841mm;
height: calc(1189mm *  (1 - 0.090 - 0.084 - 0.01) );
-webkit-column-count: 3; /* Chrome, Safari, Opera */
-moz-column-count: 3; /* Firefox */
column-count: 3;
-webkit-column-fill: auto;
-moz-column-fill: auto;
column-fill: auto;
column-gap: 0;
padding-left: 0cm;
padding-right: 0cm;
/* -webkit-column-rule-width: 50%;
-moz-column-rule-width: 50%;
column-rule-width: 50%; */
-webkit-column-rule-style: none;
-moz-column-rule-style: none;
column-rule-style: none;
-webkit-column-rule-color: black;
-moz-column-rule-color: black;
column-rule-color: black;
background-color: #ffffff;
font-family: Arial;
margin-top: calc(1189mm *  0.090 );
padding-top: 1em;
padding-bottom: 1em;
}
span.citation {
  color: #43384d;
  font-weight: bold;
}
a {
text-decoration: none;
color: #43384d;
}
#title {
font-size: 125pt;
text-align: left;
margin: 0;
line-height: 98%;
border-bottom: 0;
font-weight: normal;
background: 0;
}
#author {
color: #6C5B7B;
margin: 0;
line-height: 85%;
font-size: 1.17em;
}
#affiliation {
padding-top: 0.1em;
color: ;
font-style: italic;
font-size: 20px;
margin: 0;
}
sup {
color: #C06C84;
}
.affiliation sup {
font-size: 20px;
}
.author {
text-align: left;
}
.author sup {
font-size: 30px;
}
.author_extra {
color: #43384d;
margin: 0;
line-height: 85%;
font-size: 30px;
text-align: left;
}
.outer h1, h2, h3, h4, h5, h6 {
text-align: center;
margin: 0;
font-weight: bold;
}
.section h1 {
  text-align:center;
  padding-bottom:5px;
  background:
    linear-gradient(
      to left,
      #ffffff 1%,
      #ffffff 20%,
      #6C5B7B75 33%,
      #6C5B7B 50%,
      #6C5B7B75 66%,
      #ffffff 80%,
      #ffffff 99%
    )
    left
    bottom
    #ffffff
    no-repeat;
  background-size:100% 5px ;
  margin-top: 0.5em;
  margin-bottom: 0.5em;
}
.outer h2 {
text-align: center;
}
.outer p, .level2 {
color: #000000;
}
.outer ol {
padding-left: 8%;
padding-right: 8%;
text-align: left;
}
.main {
width: 841mm;
height: calc(1189mm * 0.090);
position: absolute;
background-color: #6C5B7B;
color: #ffffff90;
font-family: Arial;
background-image: linear-gradient(#6C5B7B 50%,#43384d);
}
.main strong {
color: #ffffff;
}
.main strong > sup {
color: #ffffff;
}
.main sup {
color: #ffffff90;
}
#main-img-left {
width: 10%;
left: 0.5in;
bottom: 0.2in;
position: absolute;
}
#main-img-center {
width: 10%;
left: calc(841mm * 0.45);
bottom: 0.5in;
position: absolute;
}
#main-img-right {
width: 10%;
right: 0.5in;
bottom: 0.2in;
position: absolute;
}
.main p {
font-size: 130px;
font-family: Arial;
text-align: center;
margin: 0;
position: absolute;
top: 50%;
-ms-transform: translateY(-50%);
transform: translateY(-50%);
margin-left: 1em;
}
.fab {
color: #00000030;
font-size: 25px;
}
.twitter, i {
color: #00000030;
font-size: 35px;
text-decoration: none;
}
a.email {
text-decoration: none;
color: #00000030;
font-size: 35px;
}
.envelope {
color: #00000030;
font-size: 5px;
text-decoration: none;
}
.poster_wrap {
width: 841mm;
height: 1189mm;
padding: 0cm;
}
.main_bottom {
width: 841mm;
height: calc(1189mm * 0.084);
margin-top: calc(1189mm * (1 - 0.084));
position: absolute;
background-color: #6C5B7B;
background-image: linear-gradient(#43384d 10%, #6C5B7B);
}
.section {
  padding-left: 10mm;
  padding-right: 10mm;
}
span > #tab:mytable {
  font-weight: bold;
}
.orcid img {
  width: 3%;
}
.emphasis {
  background-color: #008080;
  color: #ffffff;
  border: solid #0b2045 3mm;
  margin: 1em;
  padding-left: 0;
  padding-right: 0;
}
.emphasis h1 {
  font-weight: bold;
  background: none;
  background-color: #0b2045;
  padding-bottom: 5mm;
  padding-top: 1mm;
  margin-top: -1mm;
  margin-right: -1mm;
  margin-left: -1mm;
}
.emphasis blockquote {
  border: 0;
}
.emphasis ol {
  padding: 0;
  padding-left: 8%;
  font-size: 100%;
  font-weight: bold;
}
.emphasis p {
  color: #ffffff;
}
</style>
</head>
<body>


<div class="poster_wrap">

<div class="column outer">
<div class="section">

<h3 id="author" class="author">

<strong>Gihawi, A.</strong><sup> 1, <a class="orcid" href="https://orcid.org/0000-0002-3676-5561"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup><br>
<a class="twitter" href="https://mobile.twitter.com/AbrahamGihawi"><i class="fab fa-twitter"></i>&nbsp;&nbsp;@AbrahamGihawi</a><br>

<a class='envelope'><i class="fas fa-envelope"></i></a> <a href="mailto:A.Gihawi@uea.ac.uk" class="email">A.Gihawi@uea.ac.uk</a> <br>
     </h3>

<h5 id="author_extra", class="author_extra">
 Hurst, R.<sup>1</sup>
 Leggett, R.M.<sup>2</sup>
 Cooper, C.S.<sup>1</sup>
 Brewer, D.S.<sup>1,2</sup>
 Genomics England Research Consortium<sup>3</sup>
</h5>


<p id="affiliation" class="affiliation">
<sup>1</sup> Bob Champion Research and Education Building, University of East Anglia, Norwich, UK<br> <sup>2</sup> Earlham Institute, Norwich, UK<br> <sup>3</sup> Genomics England, London, UK
</p>
</div>

<div id="background" class="section level1">
<h1>Background</h1>
<p>Microbial infections cause 15% of cancers globally <span class="citation">[<a href="#ref-RN102" role="doc-biblioref">1</a>]</span>. The most significant oncomicrobes are <em>Helicobacter pylori</em> that causes gastric carcinoma <span class="citation">[<a href="#ref-RN102" role="doc-biblioref">1</a>]</span> and Human papillomavirus (HPV) that causes several types of cancer. HPV vaccination has been successful in preventing infection and tumourigenesis worldwide<span class="citation">[<a href="#ref-RN138" role="doc-biblioref">2</a>]</span>. As we transition into an era of genomic medicine, it is becoming increasingly possible to characterise the tumour-microbe landscape by interrogating high-throughput cancer sequencing data. We have developed a benchmarked approach to report evidence for microbial constituents found within human tissue samples (<a href="https://github.com/UEA-Cancer-Genetics-Lab/sepath_tool_UEA">SEPATH</a> <span class="citation">[<a href="#ref-RN454" role="doc-biblioref">3</a>]</span>). This pipeline has been applied to cancer sequences within Genomics England’s 100,000 Genomes Project. A wide range of microbial taxa are detected and here we report on the potential translational opportunities that tumour-microbial data offers.</p>
</div>
<div id="methods" class="section level1">
<h1>Methods</h1>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:methodsflow"></span>
<img src="plots/gel_flow.png" alt="SEPATH was applied to tumour whole genome sequences (WGS, _N_=11,735) from Genomics England's 100,000 Genomes Project. This pipeline extracts high quality reads that are unaligned to the human genome, runs additional human read removal and they are subject to taxonomic classification. A total of 1,617 microbial genera were detected in the cohort. The final matrix was filtered to retain dominant tumour types that were PCR Free, from fresh-frozen, primary tisue (_n_=8,909)" width="100%" />
<p class="caption">
Figure 1: SEPATH was applied to tumour whole genome sequences (WGS, <em>N</em>=11,735) from Genomics England’s 100,000 Genomes Project. This pipeline extracts high quality reads that are unaligned to the human genome, runs additional human read removal and they are subject to taxonomic classification. A total of 1,617 microbial genera were detected in the cohort. The final matrix was filtered to retain dominant tumour types that were PCR Free, from fresh-frozen, primary tisue (<em>n</em>=8,909)
</p>
</div>
<p><br></p>
<div id="pan-cancer-structure" class="section level4">
<h4>Pan-Cancer Structure</h4>
<p>A feature selection algorithm (<a href="https://cran.r-project.org/web/packages/Boruta/Boruta.pdf">Boruta</a>) identified microbial genera (495) most indicative of tumour type. A distance matrix was constructed using the Spearman’s distance with the <code>distanceMatrix</code> function (<a href="https://cran.r-project.org/web/packages/ClassDiscovery/index.html">ClassDiscovery</a> package). The distance matrix was subject to <em>t</em>-SNE (<a href="https://cran.r-project.org/web/packages/Rtsne/Rtsne.pdf">Rtsne</a> package) with parameters: <code>dims=2, perplexity=80, max_iter=2000, check_duplicates=TRUE.</code></p>
<p><br></p>
</div>
<div id="mutation-calling" class="section level4">
<h4>Mutation Calling</h4>
<p>Somatic variant VCF files were generated by Genomics England. Non-synonymous <em>TP53</em> variants of moderate-high impact were investigated in the oral/oropharyngeal cohort. Samples with no variants were considered wild-type.</p>
<p><br></p>
</div>
<div id="htlv-1-identification" class="section level4">
<h4>HTLV-1 Identification</h4>
<p>Deltaretrovirus sequencing reads from one participant were extracted from both tumour and germline samples and were subject to a Megablast search (nr/nt collection). Reads were aligned to a HTLV-1 reference <a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000863585.1/">NC_001436.1</a> using BWA-MEM with default parameters <span class="citation">[<a href="#ref-Li2009" role="doc-biblioref">4</a>]</span>. The resulting alignment was viewed with <a href="https://igv.org/">IGV</a> (version 2.9.4) <span class="citation">[<a href="#ref-Robinson2011" role="doc-biblioref">5</a>]</span>.</p>
<p><br></p>
</div>
</div>
<div id="results" class="section level1">
<h1>Results</h1>
<div id="pan-cancer-structure-1" class="section level4">
<h4>Pan-Cancer Structure</h4>
<p>Oral and colorectal tumours contain distinctive microbial elements compared to other cancer types (fig. <a href="#fig:tsnefig">2</a>).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:tsnefig"></span>
<img src="plots/kraken_tsne.png" alt="*t*-SNE plot demonstrating microbial structure for *n*=8,103 tumour samples that were subject to whole genome sequencing. Oral and Colorectal cancers typically contain microbial elements that distinguish them from other tumour types." width="100%" />
<p class="caption">
Figure 2: <em>t</em>-SNE plot demonstrating microbial structure for <em>n</em>=8,103 tumour samples that were subject to whole genome sequencing. Oral and Colorectal cancers typically contain microbial elements that distinguish them from other tumour types.
</p>
</div>
<p><br></p>
</div>
<div id="alphapapillomavirus" class="section level4">
<h4>Alphapapillomavirus</h4>
<p>Detection of Alphapapillomavirus overlaps with diagnostic tests for HPV and aligns with somatic <em>TP53</em> mutational status (fig. <a href="#fig:hpvfig">3</a>). 24/38 HPV<span class="math inline">\(^{-}\)</span> tumours contain non-synonymous <em>TP53</em> variants. All 10 HPV<span class="math inline">\(^{+}\)</span> tumours have wild-type <em>TP53</em> as expected <span class="citation">[<a href="#ref-Shi2021" role="doc-biblioref">6</a>]</span>. Alphapapillomavirus was detectable in both primary as well as metastatic tumour samples.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:hpvfig"></span>
<img src="plots/hpv_tp53.png" alt="Alphapapillomavirus detection in Oral Cancer with *TP53* mutational status." width="100%" />
<p class="caption">
Figure 3: Alphapapillomavirus detection in Oral Cancer with <em>TP53</em> mutational status.
</p>
</div>
</div>
<div id="htlv-1-detection" class="section level4">
<h4>HTLV-1 Detection</h4>
<p>We found evidence for Deltaretrovirus in one participant. A similar result was observed in the participant’s germline sample. Classified reads were extracted and uniquely matched HTLV-1 upon a <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a> search. Furthermore, the classified sequencing reads align along the length of a HTLV-1 reference genome providing additional evidence for detection (fig. <a href="#fig:htlvfig">4</a>).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:htlvfig"></span>
<img src="plots/htlv_align.png" alt="Alignment of 172 sequencing reads from a participants tumour and germline samples across a HTLV-1 reference genome (8,462 base pairs)." width="100%" />
<p class="caption">
Figure 4: Alignment of 172 sequencing reads from a participants tumour and germline samples across a HTLV-1 reference genome (8,462 base pairs).
</p>
</div>
</div>
</div>
<div id="conclusions" class="section level1">
<h1>Conclusions</h1>
<ul>
<li>Microbial nucleic acids are detectable within tumour whole genome sequence data.</li>
<li>Colorectal and oral cancers typically contain distinctive microbial elements.</li>
<li>Detection of HPV is comparable to the clinical standard. This may provide insights into 101 samples currently with no HPV test result, 8 of which are estimated positive.</li>
<li>It might be possible to detect infectious diseases like HTLV-1 but this needs to be compared to a diagnostic standard.</li>
</ul>
</div>
<div id="acknowledgements" class="section level1">
<h1>Acknowledgements</h1>
<p>Thanks to participants, staff and to <a href="http://www.big-c.co.uk/">Big C</a> for making this research possible. This research was made possible through access to the data and findings generated by the 100,000 Genomes Project; <a href="http://www.genomicsengland.co.uk">http://www.genomicsengland.co.uk</a></p>
<p><img src="resources/logos.png" width="100%" style="display: block; margin: auto;" /></p>
<p><em>Note: </em> This poster was produced in R. Details and code to reproduce this poster can be found: <a href="https://github.com/Agihawi/gel_summit_2022">github.com/Agihawi/gel_summit_2022</a></p>
<div id="references" class="section level3 unnumbered">
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-RN102" class="csl-entry">
1. Plummer M, Martel C de, Vignat J, Ferlay J, Bray F, Franceschi S. Global burden of cancers attributable to infections in 2012: A synthetic analysis. The Lancet Global Health. 2016;4:e609–16.
</div>
<div id="ref-RN138" class="csl-entry">
2. Hibbitts S, Tristram A, Beer H, McRea J, Rose B, Hauke A, et al. UK population based study to predict impact of HPV vaccination. J Clin Virol. 2014;59:109–14. doi:<a href="https://doi.org/10.1016/j.jcv.2013.12.002">10.1016/j.jcv.2013.12.002</a>.
</div>
<div id="ref-RN454" class="csl-entry">
3. Gihawi A, Rallapalli G, Hurst R, Cooper CS, Leggett RM, Brewer DS. SEPATH: Benchmarking the search for pathogens in human tissue whole genome sequence data leads to template pipelines. Genome Biol. 2019;20:208. doi:<a href="https://doi.org/10.1186/s13059-019-1819-8">10.1186/s13059-019-1819-8</a>.
</div>
<div id="ref-Li2009" class="csl-entry">
4. Li H, Durbin R. Fast and accurate short read alignment with burrows-wheeler transform. Bioinformatics. 2009;25:1754–60. doi:<a href="https://doi.org/10.1093/bioinformatics/btp324">10.1093/bioinformatics/btp324</a>.
</div>
<div id="ref-Robinson2011" class="csl-entry">
5. Robinson JT, Thorvaldsdóttir H, Winckler W, Guttman M, Lander ES, Getz G, et al. Integrative genomics viewer. Nature Biotechnology. 2011;29:24–6. doi:<a href="https://doi.org/10.1038/nbt.1754">10.1038/nbt.1754</a>.
</div>
<div id="ref-Shi2021" class="csl-entry">
6. Shi C, Liu S, Tian X, Wang X, Gao P. A <span>TP</span>53 mutation model for the prediction of prognosis and therapeutic responses in head and neck squamous cell carcinoma. <span>BMC</span> Cancer. 2021;21. doi:<a href="https://doi.org/10.1186/s12885-021-08765-w">10.1186/s12885-021-08765-w</a>.
</div>
</div>
</div>
</div>

</div>
<div class="main">
<p><strong>Translational Opportunities of Detecting Microbes in Cancer Whole Genome Sequences</strong></p>
</div>
<div class="main_bottom">
<img id="main-img-left" src=>
<img id="main-img-center" src=>
<img id="main-img-right" src=resources/poster_qr.png>
</div>
</div>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
var script = document.createElement("script");
script.type = "text/javascript";
var src = "true";
if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
if (location.protocol !== "file:" && /^https?:/.test(src))
src = src.replace(/^https?:/, '');
script.src = src;
document.getElementsByTagName("head")[0].appendChild(script);
})();
</script>


</body>
</html>
